client/server.js

```diff
@@ -1,1122 +0,0 @@
-const jsonServer = require('json-server')
```

Comment:
I deleted this file because at one time it was a useful mock backend server you could run in node to test the front end but its very outdated now and crashes the front-end because it returns invalid data.